### PR TITLE
RF simple2 query for more exhaustively defined sorting, tuned up formatting of .sparql, new produced output

### PR DESCRIPTION
**Pull request by yarikoptic**
I have noted in
- https://github.com/dbkeator/simple2_NIDM_examples/pull/16#issuecomment-1937140339
that current output is different from what was recorded in that file before. So decided to reflect that in git. Meanwhile tuned up sparql file just a little -- defined common prefix for `http://uri.interlex.org/base/` and then just adjusted formatting.

### CODE REVIEW

**Comment by yarikoptic** [comment]
now that we moved and I acquired super-powers I will try to push this effort forward by accepting my own PRs if left without statement of showstoppers from requested reviewers. For this one - I would appreciate reviews by this Fri March 1st.  Invited @satra and @dbkeator but I would welcome all @ReproNim/trd1 folks and @dnkennedy as well ;-)

**Comment by satra** [comment]
@yarikoptic - the query changes look cosmetic, so fine with me as long as it runs and produces proper output. is the csv based on data in reprolake? or some local store?

**Comment by yarikoptic** [comment]
Thank you @satra!  I think it produces the same (besides order is different as intended).
As for different output values altogether -- it is based on local population as done with what is used/modified in 
- https://github.com/ReproNim/simple2_NIDM_examples/pull/16

Is there an endpoint I could expect to run the query against and get old result???

**Comment by dbkeator** [comment]
Hi,
The CSV output file might be quite old.  I never cared about updating it.  The wBrainVols branch should contain the same NIDM files as the current version of ReproLake.  

The Readme.md file here shows how to upload the NIDM files into a local Blazegraph instance and do the queries.  We haven't written up instructions on querying ReproLake but I think @jgrethe could probably give some instruction.  I'm not entirely sure how all this will scale as we change NIDM files here and/or change what's in ReproLake, the query returns may get out of sync.  This repo was originally just a scratch space for testing, development, etc.  The ReproLake was our definitive externally-facing NIDM store....or at least that's how I always thought about it....

File: queries/simple2_query.sparql [hunk 1]
```diff
@@ -15,66 +15,60 @@ prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
...
-            ?measure a/rdfs:subClassOf nidm:DataElement;
-                 rdfs:label ?softwareLabel;
-                 nidm:measureOf <http://uri.interlex.org/base/ilx_0112559> ;
-                 nidm:datumType <http://uri.interlex.org/base/ilx_0738276> .
-            OPTIONAL {?measure nidm:isAbout ?federatedLabel }.
-            OPTIONAL {?measure nidm:hasLaterality ?laterality }.
+  ?measure a/rdfs:subClassOf nidm:DataElement;
+           rdfs:label ?softwareLabel;
+           nidm:measureOf ilxb:ilx_0112559 ;
+  nidm:datumType ilxb:ilx_0738276 .
```
Comment (surchs) [inline queries/simple2_query.sparql:25]: more indentation would help with readability here

**Comment by djarecka** [comment]
@yarikoptic - I'm testing oxigraph again on the query that Jeff gave and now on this, but I don't have any reasonable times (killing after running for hours without any results), but I'm looking at your script to do it this more automatically

**Comment by surchs** [comment]
Hey folks, I just found this site with some benchmarks of RDF stores: https://github.com/ad-freiburg/qlever/wiki/QLever-performance-evaluation-and-comparison-to-other-SPARQL-engines looks very cool - and somewhat tracks our experiences. That said: the doc is made by the maintainers of one of the tools listed (QLever)

**Comment by yarikoptic** [comment]
Let's consider this superseded by 
- https://github.com/ReproNim/simple2_NIDM_examples/pull/19

**Pull request closed without merging**